### PR TITLE
added another quote from Unit Testing book to AAA section

### PR DESCRIPTION
**Pull request by rluvaton**

### CODE REVIEW

File: sections/testingandquality/aaa.md [hunk 1]
```diff
@@ -59,3 +59,8 @@ test('Should be classified as premium', () => {
 From the book [XUnit Patterns](http://xunitpatterns.com/Four%20Phase%20Test.html):
 
 > It is important for the test reader to be able to quickly determine what behavior the test is verifying. It can be very confusing when various behaviors of the system under test (SUT) are being invoked, some to set up the pre-test state (fixture) of the SUT, others to exercise the SUT and yet others to verify the post-test state of the SUT. Clearly identifying the four phases makes the intent of the test much easier to see.
+
+
+### "Once you get used to this pattern, you can read and understand the tests more easily"
```
Comment (rluvaton) [inline sections/testingandquality/aaa.md:64]: What do you say about the title? I was debating between this and:

"_reduces the maintenance cost for your entire test suite_"

Comment (goldbergyoni) [reply sections/testingandquality/aaa.md:64]: I love the idea of adding a quote to cystalize this idea. The one you just brought is good.

The classic book for test patterns, is x-unit, this is what they have on this:

http://xunitpatterns.com/Four%20Phase%20Test.html

**Review by goldbergyoni — APPROVED** [review]
Awesome

File: sections/testingandquality/aaa.md (hunk 1 above)
Comment (rluvaton) [reply sections/testingandquality/aaa.md:64]: @goldbergyoni I don't understand, is there something you what me to change?

Comment (goldbergyoni) [reply sections/testingandquality/aaa.md:64]: No, all good

**Pull request merged**